### PR TITLE
Add fastai to libraries

### PR DESCRIPTION
**Pull request by omarespejel**
## Description
- Add fastai to the libraries in [model-libraries.md](https://github.com/huggingface/hub-docs/blob/main/docs/hub/models-libraries.md).
- Change fastai name from Fast.ai to fastai in [_toctree.yml](https://github.com/huggingface/hub-docs/blob/main/docs/hub/_toctree.yml) and [fastai.md](https://github.com/huggingface/hub-docs/blob/main/docs/hub/fastai.md).

### CODE REVIEW

File: docs/hub/models-libraries.md [hunk 1]
```diff
@@ -4,25 +4,25 @@ The Hub has support for dozens of libraries in the Open Source ecosystem. Thanks
...
+| [Speechbrain](https://speechbrain.github.io/)                               | A PyTorch Powered Speech Toolkit.                                                    | ✅             |       ✅ | ✅                 | ❌           |
+| [TensorFlowTTS](https://github.com/TensorSpeech/TensorFlowTTS)              | Real-time state-of-the-art speech synthesis architectures.                           | ❌             |       ❌ | ✅                 | ❌           |
+| [Timm](https://github.com/rwightman/pytorch-image-models)                   | Collection of image models, scripts, pretrained weights, etc.                        | ❌             |       ❌ | ✅                 | ❌           |
+| [Stable-Baselines3](https://github.com/DLR-RM/stable-baselines3)            | Set of reliable implementations of deep reinforcement learning algorithms in PyTorch | ❌             |       ✅ | ✅                 | ✅           |
+| [fastai](https://github.com/fastai/fastai)                                  | Library to train fast and accurate models with state-of-the-art outputs.             | ✅             | ✅       | ✅                 | ✅           |
```
Comment (osanseviero) [inline docs/hub/models-libraries.md:25]: We can keep alphabetical order here :smile:

**Review by osanseviero — APPROVED** [review]
LGTM! I left one suggestion for the table

File: docs/hub/models-libraries.md (hunk 1 above)
Comment (omarespejel) [reply docs/hub/models-libraries.md]: Done 🤗. I also moved sb3 to follow alphabetical order.

**Comment by omarespejel** [comment]
Thanks for the feedback @osanseviero! May I merge? :)

**Comment by osanseviero** [comment]
Thank you!

**Pull request merged**